packages/components/src/RBACProvider/RBACProvider.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { Access } from '@redhat-cloud-services/rbac-client';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+
+import {
+  RBACContext,
+  doesHavePermissions,
+  getRBAC,
+  hasAllPermissions,
+  initialPermissions,
+} from '@redhat-cloud-services/frontend-components-utilities/RBAC';
+
+const hasAccessWithUserPermissions = (userPermissions: Access[]) => {
```

Comment:
The type should be `(Access | string)[]`

Reply:
Updated.